librz/core/cmd/cmd_analysis.c

```diff
@@ -430,6 +430,31 @@ static const char *help_msg_as[] = {
 	NULL
 };
 
+/**
+ * \brief Helper to get function in \p offset
+ *
+ * Case of overlapped functions is treated as an error.
+ */
+static RzAnalysisFunction *analysis_get_function_in(RzAnalysis *analysis, ut64 offset) {
+	RzAnalysisFunction *fcn = NULL;
+	RzList *list = rz_analysis_get_functions_in(analysis, offset);
```

Comment:
Better to first try `rz_analysis_get_function_at()` before falling back to `_in` here (or if the list contains multiple entries and there is one function whose entrypoint is exactly `offset`, pick this instead of erroring, which would be equivalent).
This way, even for overlapping functions, these commands will still work if you execute them at the exact entrypoints like `afb @ fcb.ulumulu`, since entrypoints are unique.
Then in the `RZ_LOG_ERROR("Overlapped functions found\n");` below, you could also print the hint that running the code at the entrypoint of one of the functions will disambiguate.

Reply:
Done except for the hint. I just don't know how to formulate it briefly.

Reply:
"Overlapping functions found at 0x%08x. Re-run this command at the entrypoint of one of the functions to disambiguate."